content/pt-br/event-driven-architecture.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Arquitetura Orientadas por Eventos
```

Comment:
Minor: "Orientada" instead of "Orientadas"

Reply:
Feito! Muito obrigado